Sources/HipMobileUI/ViewModels/Views/AchievementsScreenViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using PaderbornUniversity.SILab.Hip.Mobile.Shared.Helpers;
+using PaderbornUniversity.SILab.Hip.Mobile.UI.Resources;
+using PaderbornUniversity.SILab.Hip.Mobile.UI.ViewModels.Pages;
+using Xamarin.Forms;
+
+namespace PaderbornUniversity.SILab.Hip.Mobile.UI.ViewModels.Views
+{
+    public class AchievementsScreenViewModel : NavigationViewModel
+    {
+        private readonly MainPageViewModel mainPageViewModel;
+
+public AchievementsScreenViewModel()
+        {
+			//mainPageViewModel = mainPageVm;
+			//need to make changes so that achievements screen asks user to log in when no user logged in
+			//if (Settings.IsLoggedIn == false)
+				
+
```

Comment:
Please remove duplicate empty lines in this file. There shouldn't be >= 2 consecutive empty lines.